src/protocols/ethereum/libs/EtherscanService.ts

```diff
@@ -46,15 +71,108 @@ export class EtherscanService {
       endblock: '99999999',
       page,
       offset,
-      sort: 'asc',
+      sort: 'desc',
+    });
+
+    if (response?.status !== '1') {
+      throw new Error(response?.result || response?.message);
+    }
+
+    return response?.result?.map(
+      (transaction: any) => EtherscanService.normalizeEtherscanTransactionStructure(
+        transaction,
+        address,
+      ),
+    ) || [];
+  }
+
+  /**
+   * Fetch ERC-20 single token or all tokens transactions.
+   */
+  async fetchAccountTokenTransactions(
+    address: string,
+    options: {
+      page?: string;
+      offset?: string;
+      /** Decides if returned transaction list will contain all ERC-20 tokens or one specified */
+      assetContractId?: AssetContractId;
+    } = {},
+  ): Promise<ITransaction[]> {
+    const {
+      assetContractId,
+      page = '1',
+      offset = TXS_PER_PAGE.toString(),
+    } = options;
+
+    const response = await this.fetchFromApi({
+      module: 'account',
+      action: 'tokentx',
+      contractaddress: assetContractId!,
+      address,
+      startblock: '0',
```

Comment:
Do we actually need to define `startblock` and `endblock` to get all the rage? Can we just omit them in the query?

Reply:
Makes sense. I just copied the code from the documentation but I see it works fine without these properties. Removed...